electronics/electric_power.py

```diff
@@ -0,0 +1,33 @@
+# https://en.m.wikipedia.org/wiki/Electric_power
+
+
+def electric_power(voltage: float, current: float, power: float) -> float:
+    """
+    This function can calculate any one of the three (voltage, current, power),
+    fundamental value of electrical system.
+    examples are below:
+    >>> electric_power(voltage=0, current=2, power=5)
+    {'voltage': 2.5}
+    >>> electric_power(voltage=2, current=2, power=0)
+    {'power': 4.0}
+    >>> electric_power(voltage=-2, current=3, power=0)
+    {'power': 6.0}
```

Comment:
Need to test:
1. No zeros
2. Two zeros
3. Negative numbers for power
4. Floating-point numbers
```suggestion
    >>> electric_power(voltage=-2, current=3, power=0).name
    'power'
    >>> electric_power(voltage=-2, current=3, power=0).value
    6.0
```

Reply:
Okay, should i also do this on ohms_law as well??

Reply:
Let's get this PR landed ___before___ you modify Ohm's law...

Coincidentally, @rhettinger (who wrote the namedtuple) just tweeted about Ohm's law...
https://twitter.com/raymondh/status/1331850115674345473